### PR TITLE
feat(MenuToggle): Add full width modifier

### LINKED ISSUE #7231: MenuToggle - add full width flag
conditionally use `pf-m-full-width` modifier

### PR DESCRIPTION
**Pull request by nicolethoen**
**What**: Closes #7231

### CODE REVIEW

File: packages/react-core/src/components/MenuToggle/examples/MenuToggle.md [hunk 1]
```diff
@@ -177,3 +177,19 @@ import { MenuToggle } from '@patternfly/react-core';
...
+
+### Full width
+
+```ts
+import React from 'react';
+import { MenuToggle } from '@patternfly/react-core';
+
+const fullWidth: React.FunctionComponent = () => {
+  return (
+    <MenuToggle isFullWidth aria-label = "Full width menu toggle" >
```
Comment (mcoker) [inline packages/react-core/src/components/MenuToggle/examples/MenuToggle.md:189]:
```suggestion
    <MenuToggle isFullWidth aria-label="Full width menu toggle" >
```

**Review by mcoker — CHANGES_REQUESTED** [review]
just one nit

**Review by thatblindgeye — APPROVED** [review]

**Review by jenny-s51 — APPROVED** [review]

**Review by mcoker — APPROVED** [review]
👏

File: packages/react-core/src/components/MenuToggle/MenuToggle.tsx [hunk 2]
```diff
@@ -16,6 +16,8 @@ export interface MenuToggleProps
   isDisabled?: boolean;
   /** Flag indicating the toggle is full height */
   isFullHeight?: boolean;
+  /** Flag indicating the toggle is takes up the full width of its parent */
```
Comment (tlabaj) [inline packages/react-core/src/components/MenuToggle/MenuToggle.tsx:19]:
```suggestion
  /** Flag indicating the toggle takes up the full width of its parent */
```

**Review by tlabaj — CHANGES_REQUESTED** [review]

**Review by wise-king-sullyman — APPROVED** [review]

**Review by kmcfaul — APPROVED** [review]

**Review by tlabaj — APPROVED** [review]

**Pull request merged**